x-pack/test/functional/apps/transform/group1/config.ts

```diff
@@ -8,13 +8,13 @@
 import { FtrConfigProviderContext } from '@kbn/test';
 
 export default async function ({ readConfigFile }: FtrConfigProviderContext) {
-  const functionalConfig = await readConfigFile(require.resolve('../../config.base.js'));
+  const functionalConfig = await readConfigFile(require.resolve('../../../config.base.js'));
 
   return {
     ...functionalConfig.getAll(),
     testFiles: [require.resolve('.')],
     junit: {
-      reportName: 'Chrome X-Pack UI Functional Tests - Transform',
+      reportName: 'Chrome X-Pack UI Functional Tests - transform - group1',
```

Comment:
I'm not a big fan of `group x` naming as it makes it harder to find/associate specific tests. When we broke down ML functional tests in the past, I applied group names that actually told you something about what the contained tests cover. I understand that it's not always possible to split tests like this such that groups still make sense. But maybe we could rename the groups slightly, e.g. like this:
* `group1` and `group4` -> `creation` directory with sub-configs:
  * `index_pattern`
  * `runtime_mappings_saved_search`
* `group2` -> `start_reset_delete`
* `group3` -> `edit_clone`

What do you think?

Reply:
I totally see your point and will rename it. I think "group x" was chosen for rebalancing post-analysis, since it is unclear how to put the test and often rebalancing is needed after smb added a new test to some of the groups.

But I don't see new tests added here for a while, so these group name should be relevant for some time.

Reply:
ee9b850f0eb4c2cb2fda73969298488f682d63c1